.github/workflows/snyk-security.yaml

```diff
@@ -0,0 +1,41 @@
+name: Snyk Security
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  snyk-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+            submodules: recursive
+
```

Comment:
```suggestion
- uses: actions/setup-node@v4
```
I think adding something like this would add caching to [npm dependencies](https://github.com/actions/setup-node#caching-global-packages-data)

Reply:
I've tried using this or similar tools, but it seems that we would then require to have Snyk CLI in a `package.json` file which would be used to compare the hash. We have a `package.json` in our ui-tests directory but it's specific to `playwright` dependencies. What do you think?

Reply:
I think it's probably fine then. It would probably only save a few moments